### PR TITLE
Add Icon Font Mixin

### PR DESCRIPTION
**Pull request by johncionci**
Regarding https://github.com/oomphinc/scss-scaffold/issues/32

Added a basic Icon Font Mixin to simplify continuously writing out `::before` & `::after`

### CODE REVIEW

File: base/_mixins.scss [hunk 1]
```diff
@@ -222,8 +222,32 @@
...
+		font-family: 'Custom Font Family';
+		font-style: normal;
+		font-weight: normal;
+		text-decoration: inherit;
+		margin-right: .5em;
+		-webkit-font-smoothing: antialiased;
+		moz-osx-font-smoothing: grayscale;
+		@if $position == after {
+			margin-left: .5em;
+		}
```
Comment (nicholemaureen) [inline base/_mixins.scss:255]: So `after`s get margins on both sides, but `before`s only get it on the right? 😛

**Review by nicholemaureen — COMMENTED** [review]
1 nitpick

File: base/_mixins.scss (hunk 1 above)
Comment (johncionci) [reply base/_mixins.scss:255]: touche

**Comment by nicholemaureen** [comment]
⛄️  /approved

**Pull request merged**